fe/fe-core/src/main/java/com/starrocks/persist/Storage.java

```diff
@@ -86,9 +87,9 @@ public void reload() throws IOException {
         if (versionFile.isFile()) {
             try (FileInputStream in = new FileInputStream(versionFile)) {
                 prop.load(in);
-                clusterID = Integer.parseInt(prop.getProperty(CLUSTER_ID));
+                versionFileMap.put(CLUSTER_ID, prop.getProperty(CLUSTER_ID));
```

Comment:
Since we already use java properties to load/store, why not use it in memory?

Reply:
I think using map to store kv is more in line with everyone's programming habits. java property is just used to write files